File: sdkwork-backend-sdk-java/src/main/java/com/sdkwork/backend/model/TeamAgentMembers.java
package com.sdkwork.backend.model;

public class TeamAgentMembers {
    private Integer id;
    private String uuid;
    private String type;
    private String bizType;
    private String description;
    private List<Object> agents;

    public Integer getId() {
        return this.id;
    }
    
    public void setId(Integer id) {
        this.id = id;
    }

    public String getUuid() {
        return this.uuid;
    }
    
    public void setUuid(String uuid) {
        this.uuid = uuid;
    }

    public String getType() {
        return this.type;
    }
    
    public void setType(String type) {
        this.type = type;
    }

    public String getBizType() {
        return this.bizType;
    }
    
    public void setBizType(String bizType) {
        this.bizType = bizType;
    }

    public String getDescription() {
        return this.description;
    }
    
    public void setDescription(String description) {
        this.description = description;
    }

    public List<Object> getAgents() {
        return this.agents;
    }
    
    public void setAgents(List<Object> agents) {
        this.agents = agents;
    }
}

File: sdkwork-backend-sdk-csharp/Models/TeamAgentMembers.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class TeamAgentMembers
    {
        public int? Id { get; set; }
        public string? Uuid { get; set; }
        public string? Type { get; set; }
        public string? BizType { get; set; }
        public string? Description { get; set; }
        public List<object>? Agents { get; set; }
    }
}
